### DISCUSSION TITLE
Modify layouts.

### DISCUSSION THREAD

**Discussion by BorjaAB** [cotes2020/jekyll-theme-chirpy#1733] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hi, I did started the [chirpy-starter](https://github.com/cotes2020/), is there any way to change the layaout if there is no _layouts folder? For example center this icons.
![imagen](https://github.com/cotes2020/jekyll-theme-chirpy/assets/26765520/4065c285-a6a6-4924-965f-29cdc5056702)

### Operations you have already tried

Search on google.

**Comment by kungfux** [comment, marked as answer]
Hi @BorjaAB.

It's possible to change the layout by overriding the existing one. To do this, you need to create a file e.g. `_layouts/post.html` that will take effect instead of the predefined one with the same filename. However, this option has its drawbacks, as you will have to manually support any future changes to stay in sync with the theme files.

So, for simple UI changes that can be achieved using CSS styles, the option to add your own styles to `assets/css/jekyll-theme-chirpy.scss` is preferable.

**Reply by BorjaAB** [reply]
Thanx you, I will check it out later, could I contact you in case that It doesn't work?

**Comment by Amane-Fujimiya** [comment]
I have similar issue to this question, but I'm new to gem-based theme, so I wonder if there's any way that I can add custom layouts to chirpy?

**Reply by br34chy** [reply]
Hi, I just resolved this issue. Chirpy starter template didn't copy _layouts, _includes, and _sass to your repo directory. 

1. `bundle info --path jekyll-theme-chirpy`
2. `cd /path/to/gems/jekyll-theme-chirpy-7.0.1`
3. cp -r _layouts /path/to/repo/
4. cp -r _includes /path/to/repo/
5. cp -r _sass /path/to/repo/
6. create a custom style.scss, insert it in _sass/layout
7. add `@import 'layout/styles';` on bottom of main.scss in _sass.

cheers!
